CHANGELOG.md

```diff
@@ -11,12 +11,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added tests for `feature.align.R` ([#40](https://github.com/RECETOX/recetox-aplcms/pull/40)), and `adjust.time.R` ([#39](https://github.com/RECETOX/recetox-aplcms/pull/40))
 - added CI to repository's GitHub Actions [#45](https://github.com/RECETOX/recetox-aplcms/pull/45),[#49](https://github.com/RECETOX/recetox-aplcms/pull/49)
 - added additional test cases for hybrid [#133](https://github.com/RECETOX/recetox-aplcms/pull/133)
+- added tests and testdata for run_filter.R (#155)
 ### Changed
 - refactored `feature.align.R` [#63](https://github.com/RECETOX/recetox-aplcms/pull/63)[#88](https://github.com/RECETOX/recetox-aplcms/pull/88)[#102](https://github.com/RECETOX/recetox-aplcms/pull/102)
 - refactored `adjust.time.R` [#64](https://github.com/RECETOX/recetox-aplcms/pull/64)[#102](https://github.com/RECETOX/recetox-aplcms/pull/102)
 - refactored `find.tol.time.R` [#91](https://github.com/RECETOX/recetox-aplcms/pull/91)
 - refactored `find.turn.point.R` [#91](https://github.com/RECETOX/recetox-aplcms/pull/91)
 - refactored `proc.cdf.R` and `adaptive.bin.R` [#137](https://github.com/RECETOX/recetox-aplcms/pull/137)
+- refactored `cont.index.R` and renamed as `run_filter.R` [#155](https://github.com/RECETOX/recetox-aplcms/pull/155)
+- Updated function names `proc.cdf.R` and `extract_features.R` [#155](https://github.com/RECETOX/recetox-aplcms/pull/155)
+- refactored `cont.index.R` and renamed as `run_filter.R` [#155](https://github.com/RECETOX/recetox-aplcms/pull/155)
```

Comment:
I don't see changes of function names `proc.cdf` and `extract_features`. Did I miss anything?

```suggestion
- added tests and testdata for run_filter.R [#155](https://github.com/RECETOX/recetox-aplcms/pull/155)
### Changed
- refactored `feature.align.R` [#63](https://github.com/RECETOX/recetox-aplcms/pull/63)[#88](https://github.com/RECETOX/recetox-aplcms/pull/88)[#102](https://github.com/RECETOX/recetox-aplcms/pull/102)
- refactored `adjust.time.R` [#64](https://github.com/RECETOX/recetox-aplcms/pull/64)[#102](https://github.com/RECETOX/recetox-aplcms/pull/102)
- refactored `find.tol.time.R` [#91](https://github.com/RECETOX/recetox-aplcms/pull/91)
- refactored `find.turn.point.R` [#91](https://github.com/RECETOX/recetox-aplcms/pull/91)
- refactored `proc.cdf.R` and `adaptive.bin.R` [#137](https://github.com/RECETOX/recetox-aplcms/pull/137)
- refactored `cont.index.R` and renamed as `run_filter.R` [#155](https://github.com/RECETOX/recetox-aplcms/pull/155)
```

Reply:
I meant to say that function names were updated  inside those files (i.e. run_filter instead of cont.index). The below line fits better. 
- Updated function names inside `proc.cdf.R` and `extract_features.R
Although, since it was a minor change better to discard that line as you suggested?.

Reply:
Yeah, I think the line stating that you renamed the function implies you reflected it in the whole package.